dask_glm/logistic.py

```diff
@@ -180,8 +188,8 @@ def newton(X, y, max_iter=50, tol=1e-8):
     '''Newtons Method for Logistic Regression.'''
 
     n, p = X.shape
-    beta = np.zeros(p)
-    Xbeta = X.dot(beta)
+    beta = np.zeros(p)  # always init to zeros?
```

Comment:
For now we can always initialize to zeros, but once we have a minimally viable API we should allow for alternate initializations (random, user-defined, etc.).